dashboard/app/views/congrats/index.html.haml

```diff
@@ -1,6 +1,8 @@
 :ruby
+  require 'cdo/user_helpers'
   congrats_data = {current_user: current_user}
```

Comment:
is age not included in the current_user object?

Reply:
birthday is, but not age. Also, I learned from Poorva recently that birthday isn't really birthday, it's the date you made an account and the year of your birth calculated from the age associated with the account info at sign up. I just used the same helper that is used in the project gallery to determine age from birthday.

Reply:
👍 good to know